src/altinn-app-frontend/src/features/form/layout/index.ts

```diff
@@ -57,10 +57,34 @@ export type ComponentTypes =
   | 'Likert'
   | 'Panel';
 
-export interface IDataModelBindings {
-  [id: string]: string;
+export interface IDataModelBindingsSimple {
+  simpleBinding: string;
+}
+
+export interface IDataModelBindingsForGroup {
+  group: string;
+}
+
+/**
+ * A middle ground between group and simple bindings, a list binding can be used to
+ * store a list of primitive values, like string[].
+ */
+export interface IDataModelBindingsList {
+  list: string;
+}
+
+export interface IDataModelBindingsForAddress {
+  address: string;
+  zipCode: string;
+  postPlace: string;
 }
```

Comment:
```suggestion
export interface IDataModelBindingsForAddress {
  address: string;
  zipCode: string;
  postPlace: string;
  careOf: string;
  houseNumber: string;
}
```

Reply:
Last two should be optional then, maybe? I haven't seen those before.

Reply:
Ah, yes! Good point.